CHANGES.rst

```diff
@@ -29,6 +29,12 @@ Gamma functions:
 * ``PolyGamma``
 * ``Stieltjes``
 
+Uniform Polyhedron
+* ``UniformPolyedron``
+* ``Dodecahedron``
+* ``Octahedron``
+* ``TetraHedron``
+
```

Comment:
Maybe add icosahedron?

Reply:
Sorry, I missed that. Much of that is in the next commit ae3c7ec  but I missed something here for this to work.

Please just add PRs on top of this if you find the mistake.